lib/honeybadger/config/ruby.rb

```diff
@@ -86,6 +86,18 @@ def backtrace_filter
         get(:backtrace_filter)
       end
 
+      def before_notify(action = nil, &block)
+        hash[:before_notify] ||= []
+
+        if action
+          hash[:before_notify] << action
+        elsif block_given?
+          hash[:before_notify] << block
```

Comment:
We know we have block at this point, so we don't need to check to see if exists. Should we check its arity?

Reply:
Checking arity sounds good :+1: